src/test/groovy/net/fabricmc/loom/test/integration/LegacyProjectTest.groovy

```diff
@@ -87,12 +87,41 @@ class LegacyProjectTest extends Specification implements GradleProjectTestTrait
 	}
 
 	@Unroll
-	def "Ancient minecraft (minecraft #version)"() {
+	def "Ancient minecraft merged (minecraft #version)"() {
+		setup:
+		def gradle = gradleProject(project: "minimalBase", version: PRE_RELEASE_GRADLE)
+		gradle.buildGradle << """
+				loom {
+                    legacyMergedMinecraftJar()
+                }
+
+                dependencies {
+                    minecraft "com.mojang:minecraft:${version}"
+                    mappings loom.layered() {
+						// No names
+					}
+
+                    modImplementation "net.fabricmc:fabric-loader:0.15.7" // 0.15.2 fixed MC 1.2.5 not launching
+                }
+			"""
+
+		when:
+		def result = gradle.run(task: "configureClientLaunch")
+
+		then:
+		result.task(":configureClientLaunch").outcome == SUCCESS
```

Comment:
I think we need to test more than just configuring the client launch. We should be able to build a jar that compliles against mc code. Maybe even try running a server with this? (might be easier said than done though)